dev/fattest.simplicity/src/componenttest/rules/repeater/RepeatTestAction.java

```diff
@@ -35,4 +38,6 @@ public interface RepeatTestAction {
      * If clean up is needed to undo the setup changes after running the action then override this method.
      */
     default public void cleanup() throws Exception {}
+
+    default public void transformApplication(Path appPath, Path newAppPath, Map<String, String> transformationRulesAppend) {}
```

Comment:
Yeah I don't want this.

Reply:
I expect people to use the following method on JakartaEEAction in order to transform to a specific EE level:

```
    public static void transformApp(Path appPath, Path newAppPath, EEVersion eeVersion)
```